common/functions/Cargo.toml

```diff
@@ -38,6 +38,7 @@ hex = "0.4.3"
 base64 = "0.13.0"
 itertools = "0.10.3"
 num-format = "0.4"
+bstr = "0.2.17"
```

Comment:
what's this crate for?

Reply:
`bstr` is a byte string library. I use the `ByteSlice` trait so that we don't need to implement `s.char_indices()` on `[u8]` agin.